docs/components.d.ts

```diff
@@ -1,9 +1,11 @@
-// generated by unplugin-vue-components
-// We suggest you to commit this file into source control
+/* eslint-disable */
+// @ts-nocheck
+// Generated by unplugin-vue-components
 // Read more: https://github.com/vuejs/core/pull/3399
-import '@vue/runtime-core'
+export {}
 
-declare module '@vue/runtime-core' {
+/* prettier-ignore */
```

Comment:
This file is included in both the `.prettierignore` and the `.eslintignore` files, so there should be no need to add relevant comments here.

Reply:
True, but the file is auto-generated, we shouldn't modify it manually.

Reply:
Okay.